### PR TITLE
Update pilot-microsoft-365-from-my-custom-domain.md

### PR DESCRIPTION
**Pull request by msbemba**
Added steps to route emails from O365 to the email org using connector

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9  Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/2888

File: microsoft-365/admin/misc/pilot-microsoft-365-from-my-custom-domain.md [hunk 1]
```diff
@@ -137,10 +178,10 @@ When you complete this step, all email sent to usera@yourcompany.com and userb@y
...
 
 After they’re imported to the appropriate locations in the Microsoft 365 mailbox, the items can be accessed from any device, anywhere.
 
-When more mailboxes are involved, or if employees are not using Outlook, you can use the migration tools available in the Exchange admin center. To get started, go to Exchange admin center, and follow the directions in [Migrate Email from an IMAP Server to Exchange Online Mailboxes – we need a new article resource].
\ No newline at end of file
+When more mailboxes are involved, or if employees are not using Outlook, you can use the migration tools available in the Exchange admin center. To get started, go to Exchange admin center, and follow the directions in [Migrate Email from an IMAP Server to Exchange Online Mailboxes – we need a new article resource].
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/misc/pilot-microsoft-365-from-my-custom-domain.md]: This paragraph should really not be published as it is. We either need a link to the reference or the last sentence should be deleted.

**Review by JohanFreelancer9 — COMMENTED** [review]
Copy review - copy is fine but please see my suggestion.

File: microsoft-365/admin/misc/pilot-microsoft-365-from-my-custom-domain.md (hunk 1 above)
Comment (msbemba) [reply microsoft-365/admin/misc/pilot-microsoft-365-from-my-custom-domain.md]: Have linked a reference document. THank you.

File: microsoft-365/admin/misc/pilot-microsoft-365-from-my-custom-domain.md [hunk 2]
```diff
@@ -137,10 +178,10 @@ When you complete this step, all email sent to usera@yourcompany.com and userb@y
...
 
 After they’re imported to the appropriate locations in the Microsoft 365 mailbox, the items can be accessed from any device, anywhere.
 
-When more mailboxes are involved, or if employees are not using Outlook, you can use the migration tools available in the Exchange admin center. To get started, go to Exchange admin center, and follow the directions in [Migrate Email from an IMAP Server to Exchange Online Mailboxes – we need a new article resource].
\ No newline at end of file
+When more mailboxes are involved, or if employees are not using Outlook, you can use the migration tools available in the Exchange admin center. To get started, go to Exchange admin center, and follow the directions in [Migrate Email from an IMAP Server to Exchange Online Mailboxes](https://docs.microsoft.com/exchange/mailbox-migration/migrating-imap-mailboxes/migrating-imap-mailboxes].
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/misc/pilot-microsoft-365-from-my-custom-domain.md]:
```suggestion
When more mailboxes are involved, or if employees are not using Outlook, you can use the migration tools available in the Exchange admin center. To get started, go to Exchange admin center, and follow the directions in [Migrate Email from an IMAP Server to Exchange Online Mailboxes](https://docs.microsoft.com/exchange/mailbox-migration/migrating-imap-mailboxes/migrating-imap-mailboxes).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@pebaum Please merge this PR. Thanks

**Pull request merged**